scripts/e2e/init-template-e2e.js

```diff
@@ -131,6 +131,25 @@ async function initNewProjectFromSource(
     );
     console.log('\nDone ✅');
 
+    if (pathToLocalReactNative != null) {
+      console.log('Updating the template version to local react-native');
+      // Update template version.
+      const appPackageJsonPath = path.join(
+        '/tmp/RNTestProject',
+        'package.json',
+      );
+      const appPackageJson = JSON.parse(
+        fs.readFileSync(appPackageJsonPath, 'utf8'),
+      );
+      appPackageJson.dependencies['react-native'] =
```

Comment:
This shouldn't be a `file://` dependency, just the version so it uses Verdaccio.

Reply:
nope, we want to use the exact same version we built in CI, so we download react native and we use the local file.
Having it to point to the locally published package means that we are going to rebuild react native locally, especially for Android, and we want to avoid that.

Reply:
Also, React Native is NOT published on Verdaccio, AFAIK.